### DISCUSSION TITLE
Scheduled playlist restarting at end time

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#7020] [General]
Not sure if I can cast this out as a general discussion as it is already out there as an issue, but here goes anyways;
I  programmed a 1 hr playlist containing 4 blocks of music interspaced with jingles to start at a set time on a set day with ending set 1 hr later.
It started at the set time on the 3 occasions I tested, but at ending time, although I could see that the song queue was set to return to Autodj at the proper time, when it came up, the programmed playlist restarter at Block01 and the queue time got reset back. I had to manually deactivate the playlist and click to skip the song block for Autodj to kick back in.
Here are the settings I had applied to the playlist in question:
![One](https://github.com/AzuraCast/AzuraCast/assets/66107794/3e2283df-21fb-4e22-bcd4-6edfe0bfd3c8)
Playlist was of course active at time of test
![Two](https://github.com/AzuraCast/AzuraCast/assets/66107794/ced2d5b5-05c5-4b33-b01c-b350739d17cd)

![Three](https://github.com/AzuraCast/AzuraCast/assets/66107794/e006b111-3c9c-4817-bf44-715f3cd539b9)
Tested all options to no avail

And here is the content of the playlist;
![Four](https://github.com/AzuraCast/AzuraCast/assets/66107794/c27c8311-3a26-4021-9e49-9e96a8657e16)

Not sure what I'm doing wrong, but can't get the stream to return to Autodj at the end of programmed time... any thoughts/solutions on this?
Thanks for your help
Speedy

**Comment by gAlleb** [comment]
Try turning on `Boucler une fois` switch. This should do the trick. If you want all 4 tracks to be surely played - maybe set the ending time a little bit further to `16:30` for example. With `loop once` option `switch` would return to normal rotation after last track in a playlist has played.
Maybe it's due to playlist being 'sequential'. Or maybe a bug.

**Reply by SpeedyCharly** [reply]
Thank ... will test that tonight . Much appreciated

**Reply by SpeedyCharly** [reply]
Thanks @gAlleb - Your suggestion plus a little tweak to set times did the trick !!!!
